documentation/dev-portal/src/clients/socks5-client.md

```diff
@@ -0,0 +1,66 @@
+                                                                             |       +--------------------+
+                                                                             |       +--------------------+
+                                                                             |------>|    RPC endpoint    |
+                                                                             |       +--------------------+
+                                                                             |       +--------------------+
+                                                                             |------>|       Website      |
+                                                                             |       +--------------------+
+                                                                             |       +--------------------+
+  +----------------------------------+                                       |------>|       etc...       |
+  | Mixnet:                          |                                       |       +--------------------+
+  |       * Gateway your client is   |                                       |
+  |       connected to               |          +--------------------+       |
+  |       * Mix nodes 1 -> 3         |<-------->| Network requester  |<------+
+  |       * Gateway that network     |          +--------------------+
+  |       requester is connected to  |
+  +----------------------------------+
+           ^
+           |
+           |
+           |
+           |
+           v
+ +-------------------+
+ | +---------------+ |
+ | |  Nym client   | |
+ | +---------------+ |
+ |         ^         |
+ |         |         |
+ |         |         |
+ |         |         |
+ |         v         |
+ | +---------------+ |
+ | | Your app code | |
+ | +---------------+ |
+ +-------------------+
+  Your Local Machine
+```
+
+There are 2 pieces of software that work together to send SOCKS traffic through the mixnet: the `nym-socks5-client`, and the `nym-network-requester`.
+
+The `nym-socks5-client` allows you to do the following from your local machine:
+* Take a TCP data stream from a application that can send traffic via SOCKS5.
+* Chop up the TCP stream into multiple Sphinx packets, assigning sequence numbers to them, while leaving the TCP connection open for more data
+* Send the Sphinx packets through the mixnet to a [network requester](https://nymtech.net/operators/nodes/network-requester.md). Packets are shuffled and mixed as they transit the mixnet.
+
+The `nym-network-requester` then reassembles the original TCP stream using the packets' sequence numbers, and make the intended request. It will then chop up the response into Sphinx packets and send them back through the mixnet to your  `nym-socks5-client`. The application will then receive its data, without even noticing that it wasn't talking to a "normal" SOCKS5 proxy!
+
```

Comment:
I wonder if to add a note that somewhere most of NRs nowadays are embedded in exit gw - which is a mode of nym-node.

Reply:
Definitely!